src/main/java/xbot/common/command/BaseMaintainerCommand.java

```diff
@@ -207,11 +207,6 @@ protected boolean getErrorWithinTolerance() {
      */
     protected abstract double getHumanInputMagnitude();
 
-    @Override
-    public String getPrefix() {
-        return subsystemToMaintain.getPrefix() + getName() + "/";
```

Comment:
This causes a nullref as the subsystem can't easily be created. Maintainer records will need to live under their own command, which at least matches up with other commands.